gallery/src/components/NxLoadingSpinner/NxLoadingSpinnerExample.tsx

```diff
@@ -8,6 +8,6 @@ import React from 'react';
 
 import { NxLoadingSpinner } from '@sonatype/react-shared-components';
 
-const NxLoadingSpinnerExample = () => <NxLoadingSpinner />;
+const NxLoadingSpinnerExample = () => <NxLoadingSpinner>Foo</NxLoadingSpinner>;
```

Comment:
That you can pass children to the loading spinner is not publicly documented.  Why was it added to this example?

Reply:
> That you can pass children to the loading spinner is not publicly documented. Why was it added to this example?

I accidentally pushed that, I was just testing it out and forgot to revert my changes: 370d7c9e4538654aac4691ccc387f99bf99072ef